src/server/game/DataStores/DBCEnums.h

```diff
@@ -839,10 +839,69 @@ DEFINE_ENUM_FLAG(ChrCustomizationReqFlag);
+    CURRENCY_TYPE_THE_AVOWED                = 1884, // Grants reputation with the Avowed.
+    CURRENCY_TYPE_DEATHS_ADVANCE            = 1907, // Grants reputation with the Death's Advance.
+    CURRENCY_TYPE_THE_ENLIGHTENED           = 1982, // Grants reputation with The Enlightened.
+    CURRENCY_TYPE_ARCHIVISTS_CODEX          = 1997, // Grants progress with the Archivists' Codex.
+    CURRENCY_TYPE_DRAGON_ISLES_SUPPLIES     = 2003, // Various resources and building materials that can assist the friendly factions of the Dragon Isles in their fight against the Primalists. Used to purchase items unlocked from renown.
+    CURRENCY_TYPE_DRAGONSCALE_EXPEDITION    = 2031, // Grants Reputation with the Dragonscale Expedition.
+    CURRENCY_TYPE_VALDRAKKEN_ACCORD         = 2106, // Grants Reputation with the Valdrakken Accord.
+    CURRENCY_TYPE_MARUUK_CENTAUR            = 2108, // Grants Reputation with the Maruuk Centaur.
+    CURRENCY_TYPE_ISKAARA_TUSKARR           = 2109, // Grants Reputation with the Iskaara Tuskarr.
+    CURRENCY_TYPE_COBALT_ASSEMBLY           = 2134, // Grants Cobalt Assembly Reputation
+    CURRENCY_TYPE_LOAMM_NIFFEM              = 2420, // Grants Reputation with the Loamm Niffen.
+    CURRENCY_TYPE_SORIDORMIS_RECOGNITION    = 2645, // Grants Reputation with Soridormi.
+    CURRENCY_TYPE_DREAM_WARDENS             = 2652, // Grants Reputation with the Dream Wardens.
+    CURRENCY_TYPE_WHELPLINGS_DREAMING_CREST = 2715, // Earned from many outdoor activities, Raid Finder Amirdrassil, and Mythic Keystone dungeons up to +5 during Dragonflight Season 3.
+    CURRENCY_TYPE_DRAKES_DREAMING           = 2716, // Earned from some outdoor activities, Normal difficulty Amirdrassil, and Mythic Keystone dungeons from +6 to +10 during Dragonflight Season 3.
+    CURRENCY_TYPE_WYRMS_DREAMING            = 2717, // Earned from Heroic difficulty Amirdrassil and Mythic Keystone dungeons from +11 to +15 during Dragonflight Season 3.
+    CURRENCY_TYPE_ASPECTS_DREAMING          = 2718, // Earned from Mythic difficulty Amirdrassil and Mythic Keystone dungeons from +16 and up during Dragonflight Season 3.
+    CURRENCY_TYPE_AZEROTHIAN_ARCHIVES       = 2719, // Progress with the Azerothian Archivists.
+    CURRENCY_TYPE_COUNCIL_OF_DORNOGAL       = 2897, // Grants Reputation with the Council of Dornogal.
+    CURRENCY_TYPE_HALLOWFALL_ARATHI         = 2899, // Grants Reputation with the Hallowfall Arathi.
+    CURRENCY_TYPE_THE_ASSEMBLY_OF_THE_DEEPS = 2902, // Grants Reputation with The Assembly of the Deeps
+    CURRENCY_TYPE_THE_SEVERED_THREADS       = 2903, // Grants Reputation with The Severed Threads.
+    CURRENCY_TYPE_THE_WEAVER_NOTORIETY      = 3002, // Grants Reputation with The Weaver.
+    CURRENCY_TYPE_THE_GENERAL_NOTORIETY     = 3003, // Grants Reputation with The General.
+    CURRENCY_TYPE_THE_VIZIER_NOTORIETY      = 3004, // Grants Reputation with The Vizier.
+    CURRENCY_MAX_CAP_ANCIENT_MANA           = 4000
```

Comment:
Is the cap change intended?

Reply:
> Is the cap change intended?

I set that value because I thought it should be higher than the last one. What value do you recommend?

Reply:
> Is the cap change intended?

Do I keep the value I had?

Reply:
MaxQty is 300, so idk why it was set to 2k to begin with, unless there was another reason for that value?

Reply:
> MaxQty is 300, so idk why it was set to 2k to begin with, unless there was another reason for that value?

That's default max quantity, you can increase the cap to 2000

Reply:
> > Is the cap change intended?
> 
> Do I keep the value I had?

Yes, that value isn't a currency id, is the max quantity for ancient mana (after increasing cap)

Reply:
In that case it doesn't belong in there, that enum is for types only.

Reply:
> Yes, that value isn't a currency id, is the max quantity for ancient mana (after increasing cap)

Yes, right, I understood that and I'm updating.

Reply:
Your last change is wrong. Just keep CURRENCY_MAX_CAP_ANCIENT_MANA as it was defined (2000) and remove the rest of the caps (Player::GetCurrencyMaxQuantity takes into account required db2 fields)

Reply:
You really did not have go add the max caps for every currency type either as that can simply be retrieved from DB2...